core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -114,6 +114,10 @@ package object config {
     .checkValue(_ >= 0, "The off-heap memory size must not be negative")
     .createWithDefault(0)
 
+  private[spark] val PYSPARK_EXECUTOR_MEMORY = ConfigBuilder("spark.executor.pyspark.memory")
```

Comment:
Argh, should have noticed this before. Should this be added to `configuration.md`?